fire/helputils.py

```diff
@@ -178,8 +190,9 @@ def UsageString(component, trace=None, verbose=False):
   command = trace.GetCommand() + ' ' if trace else ''
 
   if inspect.isroutine(component) or inspect.isclass(component):
-    args, varargs, keywords, defaults = inspectutils.GetArgSpec(component)
-    return _UsageStringFromFnDetails(command, args, varargs, keywords, defaults)
+    spec = inspectutils.GetArgSpec(component)
+    spec_dict = dict(zip(spec._fields, iter(spec)))
+    return _UsageStringFromFnDetails(command, **spec_dict)
```

Comment:
in line w/ the previous comment, this would become _UsageStringFromFnDetails(command, spec).